src/plugins/apps/index.js

```diff
@@ -0,0 +1,65 @@
+//@ts-check
+const { default: fetch } = require('node-fetch');
+
+module.exports = async function appsPlugin() {
+  // TODO: actually use the electron/apps repo as a data source
+  const response = await fetch(
+    'https://raw.githubusercontent.com/erickzhao/apps/master/index.json'
+  );
+  const apps = await response.json();
+  return {
+    name: 'apps-plugin',
+    async loadContent() {
+      const FAVS = new Set([
+        '1password',
+        'asana',
+        'discord',
+        'figma',
+        'flat',
```

Comment:
```suggestion
        'agora-flat',
```
The full name of the product is `Agora Flat`

Reply:
Sorry. This is an issue I accidentally introduced earlier and I have submitted a fix PR to apps. 🙇‍♂️

Ref: https://github.com/electron/apps/pull/2049